docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/TracerV.rst

```diff
@@ -228,33 +229,22 @@ The human readable trace output format looks like so:
 .. include:: TRACERV-HUMAN-READABLE-EXAMPLE
    :code: ini
 
-In this output, C0 represents instruction stream (or core) zero. The next field
-represents the address of the instruction committed that cycle and a valid bit,
-in hex, interpreted like so:
-
-.. code-block:: ini
```

Comment:
I actually think it would still be nice to have an example of the format (even if the format has changed and is now more readable)

Reply:
Ok